src/coreclr/debug/ee/controller.cpp

```diff
@@ -7553,6 +7632,31 @@ void DebuggerStepper::TriggerUnwind(Thread *thread,
     m_reason = unwindReason;
 }
 
+void DebuggerStepper::TriggerMulticastDelegate(DELEGATEREF pDel, INT32 delegateCount)
+{
+    TraceDestination trace;
+    FramePointer fp = LEAF_MOST_FRAME;
+
+    INT_PTR totalDelegateCount = pDel->GetInvocationCount();
+    if (delegateCount == totalDelegateCount)
```

Comment:
Now that the ILStub was modified not to call the helper after the final delegate has been invoked we should never observe delegateCount == totalDelegateCount. You can delete the if statement because only the else portion ever runs.